src/bindings/python/src/openvino/runtime/__init__.py

```diff
@@ -5,6 +5,27 @@
 """openvino module namespace, exposing factory functions for all ops and other classes."""
 # noqa: F401
 
+import warnings
+warnings.simplefilter("always", DeprecationWarning)
+import inspect
+stack = inspect.stack()
+
+if stack[-1].code_context is None:  # python interactive mode
+    warnings.warn(
+        "The `openvino.runtime` module is deprecated and will be removed in the 2026.0 release. "
+        "Please replace `openvino.runtime` with `openvino`.",
+        DeprecationWarning,
+        stacklevel=2
+    )
```

Comment:
the main thing is not to clutter ovc/convert_model and nncf output logs with this warning. 
Can you please check that their output still have pure output logs?

Reply:
I checked nncf and I can see that the warning is shown once per python process, but it is triggered also by `benchmark_app`

Reply:
I think we need to fix them (can be in separate PR) just to have uncluttered output. Just ping NNCF guys, to remove usage `openvino.runtime`

Reply:
benchmark_app [fixed](https://github.com/openvinotoolkit/openvino/pull/28433)